pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -173,6 +173,12 @@
   SSL_FACTORY_ARG("sslfactoryarg", null,
       "Argument forwarded to constructor of SSLSocketFactory class."),
 
+   /**
+    * The String argument to give to the constructor of the SSL Context Factory.
+    */
+   SSL_CONTEXT_PROTOCOL("sslContextProtocol", "TLS",
+          "Argument forwarded to constructor of SSLContext. Explicit SSL protocol to use when creating an SSLContext."),
```

Comment:
Should this include choices like: "TLS", "TLSv1", "TLSv1.1", "TLSv1.2"? That will give a hint to what are the posible values.

http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#SSLContext